PySpice/Spice/HSpice/Server.py

```
@@ -0,0 +1,266 @@
+          meas_file_path = os.path.join(tmp_dir, f)
+          logger.info(f"Parsing measurement file: {meas_file_path}")
+          try:
+            with open(meas_file_path, "r") as mf:
+              lines = mf.readlines()
+            content_lines = [l.strip() for l in lines if l.strip() and not l.startswith('$') and not l.startswith('.')]
+            if len(content_lines) >= 2:
+              names = content_lines[0].split()
+              is_sweep = len(content_lines) > 2
+              for val_line in content_lines[1:]:
+                vals = val_line.split()
+                for name, val in zip(names, vals):
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)